src/merkledag.proto.js

```diff
@@ -1,3 +1,6 @@
+'use strict'
+
+module.exports = new Buffer(`
```

Comment:
@dignifiedquire my main point is: **we need to have a section in our main projects (js-ipfs, js-ipfs-api and js-libp2p) that explains our users how to 'webpack' all of this stuff**, simply, it is better to have a walkthrough, even if it is obvious than having nothing. As an example, I don't believe that js-ipfs will ever stop needing a special webpack config.